setup.iss

```diff
@@ -177,6 +177,12 @@ begin
           RemoveBOM(ExpandConstant('{app}\EXE\ReShadePreset.ini'));
         end;
 
+        // Write d3d8 DLL override for Wine/Linux. For more information, see https://wiki.winehq.org/Wine_User%27s_Guide#DLL_Overrides
+        if IsWine then
+        begin
+          RegWriteStringValue(HKEY_CURRENT_USER, 'Software\Wine\DllOverrides', 'd3d8', 'native,builtin');
```

Comment:
You can append this expression to the previous if statement by using the `else` keyword, since that already checks the opposite of `if IsWine`. This way you don't have to write another if statement.

Reply:
Thanks for the suggestion! It didn't cross my mind to use an else. Please check it now and see if you like it. I'll be glad to give you access to my fork if you want to change it directly. (It compiles and runs just fine in Wine.)